ykcompile/src/lib.rs

```diff
@@ -208,11 +192,10 @@ impl TraceCompiler {
             }
             ykpack::CallOperand::Unknown => {}
         };
-        // Save all registers to the stack.
-        self.store_registers();
         // Move call arguments into registers.
+        let local_offset = dest.as_ref().unwrap().local;
```

Comment:
Sometimes we use "offset", sometimes "idx" as a suffix. Should we be consistent?

Reply:
The things is these all describe different things:
- `Local offset` is offset for the arguments.
- `var_count` tells us how many variables we've assigned so far so we know what the next variables name is.

I feel like calling them all `idx` isn't very accurate either.

Reply:
Fair enough. I do feel like we can probably be more consistent than we currently are though. For example, `vars_len` might work better (and it's shorter than `count`) and `args_off` might work. I'm not saying those are the right names, but just lobbing a few ideas into the pot.

Reply:
Fixed in dae83bd (I didn't come up with any better names though).